lib/submodule_update_flow.rb

```diff
@@ -39,14 +40,22 @@ def candidate_prs_to_close
     end
   end
 
-  # If there are changes by other developers, don't close the PR
+  # If there are changes by other developers or if it's assigned, don't close the PR
   def prs_to_close
     candidate_prs_to_close.filter do |_, pr|
-      has_other_commits = @client_repo.pull_commits(pr.number).any? do |commit|
-        commit.author.login != 'wpmobilebot'
-      end
+      has_other_commits = pr_has_other_commits?(pr.number)
       puts "PR ##{pr.number} has commits by other developers, skipping.." if has_other_commits
-      !has_other_commits
+
+      is_assigned = !pr.assignees.empty?
+      puts "PR ##{pr.number} is assigned to #{pr.assignees.map(&:login)}, skipping.." if is_assigned
+
+      !has_other_commits && !is_assigned
+    end
+  end
+
+  def pr_has_other_commits?(pr_number)
+    @client_repo.pull_commits(pr_number).any? do |commit|
+      commit.author.login != 'wpmobilebot'
```

Comment:
nit – if we ever change the token, this might not be correct anymore. It might be better to fetch this via the API (with a `whoami` type call to the GitHub API). 

Doesn't have to be done now, just wanted to flag it.

Reply:
Yeap - there are lots of little things like that. At this point it's still not much more than a HACK week project.

If it turns out we are using it heavily and adding more features, it'll be worth spending some time to fix them all.